src/component/CreateBankAccount.js

```diff
@@ -89,6 +89,12 @@ export function CreateBankAccount() {
           </Form>
         </Card.Body>
       </Card>
+
+      <div className='w-100 text-center mt-2'>
+        <Link to='/dashboard'>
```

Comment:
There is something nice that can be done with the [history hook](https://reactrouter.com/web/api/history). Make `{history.goBack()}` would truer to the `Go back` button :). I haven't tested it yet. Just a suggestion.